src/components/Composer.vue

```diff
@@ -226,6 +226,18 @@
 			{{ t('mail', 'Retry') }}
 		</button>
 	</div>
+	<div v-else-if="state === STATES.WARNING" class="emptycontent" role="alert">
+		<h2>{{ t('mail', 'Warning sending your message') }}</h2>
+		<p v-if="errorText">
+			{{ errorText }}
```

Comment:
this won't be translated. can we check on the error type and have a special case for ``ManyRecipientsError``?

Reply:
I did this by copy pasting and adapting L217-228. Matching for `ManyRecipientsError` happens in L740.

Reply:
my bad.